### ISSUE TITLE
traffic data missing for some road_segment_id

### ISSUE THREAD

**Issue by paladin2000cn** [JingqingZ/BaiduTraffic#8]
I was reading the road_segment_id from the neighbour_1km.txt. Using the road_segment_id as key to fetch the speed value from the traffic_speed_sub-dataset. some road_segment_id can not be found in traffic_speed_sub-dataset.  for example:
read one line in neighbour_1km.txt. -> all the ids has been mapped from the link_id_hash_map
['1597566463414', '1462215565312', '1503121983912', '1770727782763', '1462073565097', '1503110983921', '1597550463401', '1682917053527', '1687870507191', '1597560463393', '1770727782763']
1503110983921 is missing in traffic_speed_sub-dataset.
we assmue that all the id in neighbour_1km.txt should be contained in the traffic_speed_sub-dataset. could you please explain that?

**Comment by bbliao** [comment]
@paladin2000cn Yes,  all the id in neighbour_1km.txt are contained in the traffic_speed_sub-dataset. And I have checked the traffic_speed_sub-dataset file, it **does** contains the link "1503110983921". Maybe you should check your code. Thank you!

**Comment by vision-zhao** [comment]
I'm checking the dataset, and I feel confused with some issues.
1. what does this file "neighbour_1km.txt" represent?the neighbour within 1km of each segment?but I can't figure out what segment each row point to?
2. it's always mentitoned in your paper that you have selected 15073 interested segments in your research, but the datasets of traffic speed all contain more than 15073 segments. so could you help me to point out which segments have you selected?
THANK YOU

**Comment by vision-zhao** [comment]
I'm checking the dataset. and I feel confused about some issues.
1. What does the file 'neighbour_1km.txt" represent? if it represents the neighbours within 1km of each segment, which segment does each rows point to?
2. all the datasets of speed/road contain more than 15073 segments, which is the number you mentioned in your paper. so which segments have you chosed for your research?
Thank you

**Comment by bbliao** [comment]
@vision-zhao As mentioned in the paper, we only predict the traffic speed of 15,073 road segments, i.e., the first column of the "neighbour_1km.txt". Since we want to consider the spatial relation in the road network, we also provide the traffic speed of the neighbors, i.e., the 2nd to 11th column of the "neighbour_1km.txt". At last, we provide 45,148 road segments, and for each road segment (column 1), all the neighbors (column 2- 11) are within 1km.
